assets/templateFloxEnv/manifest.toml

```diff
@@ -1,8 +1,8 @@
 # List packages you wish to install in your environment under
 # the `[install]` table
 # [install]
```

Comment:
suggestion: Write template with valid initial `install` table

When this is commented, flox install creates _a new_ `[install]` table at the top of the file, I think if we keep it uncommented it will use the table here as an anchor and install packages just above the examples 

Consider doing the same with `[hook]` and `[vars]` (vars are gone now?)


```suggestion
[install]
```

Reply:
Fixed the `[install]` part. We don't have a similar automatic method of adding to `vars` or `hook` so I'm not too concerned with adding those in.

Reply:
yeah i just though for consistency but no worries